src/cooperativa.py

```diff
@@ -54,3 +59,14 @@ def productos(self):
                 Productos de la cooperativa
         """
         return self._productos
+
+    @property
+    def agricultores(self):
+        """
+        Getter de agricultores
+
+        Returns:
+            agricultores: lista de enteros
+                IDs de los agricultores que son socios de la cooperativa
+        """
+        return self._agricultores
```

Comment:
¿Para qué lo haces "privado", si das acceso a él directamente como property? ¿Dónde dice además que cualquier cliente de este objeto tenga que tener acceso a los agricultores? ¿No es privado?
Conviene que hagáis absolutamente *todo* el diseño de la clase de acuerdo con las restricciones que se planteen en las historias de usuario y de acuerdo con los tickets (problemas) que se saquen de ellos.

Reply:
La lista de objetos de agricultor.py ya está modificada.